marko.json

```diff
@@ -93,5 +93,26 @@
             "@*": "expression"
         }
     },
-    "<ebay-breadcrumb-item>": {}
+    "<ebay-breadcrumb-item>": {},
+    "<ebay-pagination>": {
+        "renderer": "./src/components/ebay-pagination/index.js",
+        "transformer": "./src/common/transformer/index.js",
+        "@class": "string",
+        "@accessibility-prev": "string",
```

Comment:
We could abbreviate `accessibility` to `a11y` if everyone is in agreement. But I'm fine keeping as is.

Reply:
The notice and the dialog have `aria-label-x` attributes. Wouldn't it make sense to be consistent with those? IE: `aria-label-prev` `aria-label-next` and `aria-label-current`.

Reply:
We might want to move towards more generic naming like `accessibility` or `a11y` so we can be consistent with cases where the strings don't directly map to `aria-label`s. https://github.com/eBay/ebayui-core/pull/81/files#diff-e56f5fdf7a730e2c503dd8217ef07299R21

Reply:
Should I send a separate PR to change the naming in notice component?

Reply:
Okay I will keep the naming here as is and send a separate PR for notice